app/src/main/java/com/codesoom/assignment/models/Path.java

```diff
@@ -0,0 +1,50 @@
+package com.codesoom.assignment.models;
+
+import com.codesoom.assignment.utils.StringUtil;
+
+public class Path {
+    String path;
+    String pathVariable;
```

Comment:
이 값들을 재할당이 방지된 비밀 멤버로 수정해 보세요. 생성자 작업이 필요할 겁니다.
```suggestion
    private final String path;
    private final String pathVariable;
```

Reply:
기존 path는 최초 생성 후 ContextContainer에서 다시 setPathvariable을 하도록 설계하였습니다.
따라서 해당 작업을 (e4a0d9b) 수정 내역에서 private final 처리 후 매개변수 생성자를 통해 인스턴스가 생성되도록 수정해보았습니다!